DEV-9/File.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.IO;
+
+namespace DEV_9
+{
+    public class File
+    {
+        public string[] Read(string path, int NUMBER_OF_ROWS)
+        {
+            try
+            {
+                using (StreamReader streamReader = new StreamReader(path, System.Text.Encoding.Default))
+                {
+                    string[] arrayOfLines = new string[NUMBER_OF_ROWS];
```

Comment:
Ate you sure that file will contain NUMBER_OF_ROWS lines? What will happened if  file is empty?